### ISSUE TITLE
A11y issues

### ISSUE THREAD

**Issue by FaberVitale** [Roma-JS/roma-js-on-astro#76]
### Describe the bug

There a few a11y issues to address in the current iteration of [romajs.org](https://romajs.org/).

Most of the issues have been spotted using https://wave.webaim.org/ and Voiceover(Mac screen reader).

### Wave reports

| page | report |
|-----|------|
| hp | https://wave.webaim.org/report#/https://romajs.org/it |
| upcoming events | https://wave.webaim.org/report#/https://romajs.org/it/prossimi-eventi/ |
| prev events | https://wave.webaim.org/report#https://romajs.org/it/eventi-passati/1/  |
| about | https://wave.webaim.org/report#/https://romajs.org/it/about/ |
| blogList | https://wave.webaim.org/report#/https://romajs.org/blog/ |
| blog page | https://wave.webaim.org/report#/https://romajs.org/blog/post/2022-03-13-sessione-sulle-webperformance/ |

### Mobile Navbar

The navbar has small a11y issues when tested with the screen reader.

### Language selector

The lang selector ( it | en ) is not clear enough when using a screen reader,
the links do not have meaningful [a11y names](https://developer.mozilla.org/en-US/docs/Glossary/Accessible_name).

### bypass block

A bypass block to skip the navbar  is missing. 

See https://www.w3.org/TR/WCAG22/#bypass-blocks for more details.

### System Info

```bash
not relevant
```

### Severity

annoyance
